docusaurus/docs/quickstarts/network/hosted/index.md

```diff
@@ -1,168 +1,258 @@
+It is not necessary for the server to be on the open internet. If it works better for you to deploy OpenZiti on your
+own network, great, do that.  The only requirement to be aware of is that every piece of the a network will need to be able to communicate to the controller at least one edge router.
+
+If you have a server available on the open internet, or you will provision one for use with OpenZiti, that's the
+ideal scenario. With a zero trust overlay network provided by OpenZiti, you can rest assured that your traffic is safe even when using commodity internet. Furthermore, you do not need to worry about being on a network you trust, as all networks are considered untrustworthy, even your work/home network!
+
+## Installation
+
+When starting out deploying a [OpenZiti Network](../index.md#overview-of-a-ziti-network), we recommend you follow
+and use the `expressInstall` function provided by the OpenZiti project. Once you're familiar with the network and
+the configuration options available you'll be better equipped to make changes.
+
+### Firewall
+
+The first issue you will need to deal with is opening some ports. A network will consist of at least one controller and 
+at least one edge router. Both of these components will require ports to be open. For the controller you will need to 
+open a range of ports through your firewall:
+
+- `8440/tcp`: Edge Controller providing router control plane
+- `8441/tcp`: Edge Controller providing client sessions
+- `8442/tcp`: Edge Router providing client connections
+- `8443/tcp`: Ziti Admin Console (ZAC) [optional]
+
+These are the arbitrary ports we'll use in this example for convenience when specifying the firewall exception as a port range.
+
+## Express Install
+
+With the firewall ports open you will now be able to source the script which provides the expressInstall function and
+execute it. You probably should consider DNS names before actually running the script. By default, the script will use the output of the
+`hostname -s` command to determine a reasonable default for the PKI that will be generated as well as the
+configuration files that are output.  The script allows you to configure these values and if you are deploying to a cloud
+provider (AWS, Azure, OCI, IBM, Digital Ocean, GCP, etc.) you will almost certainly want to use the **public** DNS name
+of your instance. You possibly  want to override the IP address to use as well, but using DNS names is superior to using
+IP addresses in case your IP changes.
+
+The quickest and easiest thing to do, is simply find your external DNS name and set it into the EXTERNAL_DNS environment
+variable. For example:
+
+```bash
+export EXTERNAL_DNS="acme.example.com"
+```
+
+A random password for user "admin" will be generated and stored in the `*.env` file inside the express install home directory so that it can be easily referenced by the `zitiLogin` function. Optionally, you may override with a preferred password by setting `ZITI_PWD` now.
```

Comment:
i'm quite torn on "non-line-breaking" markdown like this. part of me likes it because moving text is easier to see, part of me dislikes it because it's not very "markdown-y". Have you found any good guidance on "the right way" to markdown?  I've been wrapping at around 120 chars (not hard, just ish) as a good page width. i'm open to NOT doing that, i would just like to be consistent.  also reviewing the doc here on GH is easier when it's wrapped ~120 ish chars

Reply:
I have also been hard wrapping at 120 for two reasons, one is that it was already being done, two is because I find it easier to read locally without needing to use a MD renderer to soft wrap.

Now that I've said that "out loud" I have to believe there is a soft wrap option in GoLand that can be set to MD only.

Reply:
And in fact there is... View-->Active Editor-->Soft-Wrap. It only applies to the currently active editor so, not all MD files but I would be fine with this workaround if we decided to go without hard line breaks.

Reply:
Glad you found that. I'm using soft wrap in my editor so hard wraps are of no immediate value to me or most readers. Let's soft wrap in our editors to ease reading and editing and optimize for the following in commits:

1. vast majority of readers are reading styled HTML, not Markdown, so hard wraps have no value to them
2. minimize visible differences in Git diff i.e. if we hard re-wrap after changing a line that creates a lot of visual noise in the diff.